src/main/java/org/syftkog/web/test/framework/Language.java

```diff
@@ -0,0 +1,191 @@
+package org.syftkog.web.test.framework;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.xml.bind.annotation.XmlEnum;
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ *
+ * @author BenjaminLimb
+ */
+@XmlType(name = "langauge")
+@XmlEnum
+public enum Language {
+
+  /**
+   *
+   */
+  DE("Deutsch", "German", "de", "deu"),
+
+  /**
+   *
+   */
+  PT("Português", "Portuguese", "pt", "por"),
+
+  /**
+   *
+   */
+  EN("English", "English", "en", "eng"),
+
+  /**
+   *
+   */
+  RU("Русский", "Russian", "ru", "rus"),
+
+  /**
+   *
+   */
+  ES("Español", "Spanish", "es", "spa"),
+
+  /**
+   *
+   */
+  JA("日本語", "Japanese", "ja", "jpn"),
+
+  /**
+   *
+   */
+  FR("Français", "French", "fr", "fra"),
+
+  /**
+   *
+   */
+  KO("한국어", "Korean", "ko", "kor"),
+
+  /**
+   *
+   */
+  IT("Italiano", "Italian", "it", "ita"),
+
+  /**
+   *
+   */
+  ZH("中文", "Chinese", "zh", "zho"),
+
+  /**
+   *
+   */
+  EO("Psuedo Loc", "Psudo Loc", "eo", "eo");
+    private static final Map<String, Language> twoLetterCodeMap = new HashMap<>();
+    private static final Map<String, Language> threeLetterCodeMap = new HashMap<>();
+
+    static {
+        for (Language lan : Language.values()) {
+            twoLetterCodeMap.put(lan.twoLetterCode, lan);
+            threeLetterCodeMap.put(lan.threeLetterCode, lan);
+        }
+    }
+
+    private Language(final String name, final String englishName, final String twoLetterCode, String threeLetterCode) {

```

Comment:
FYI: These display names in their own language as well as in English and their respective two- and three- letter codes are already available in java.util.Locale via getDisplayName(Locale) passing in Locale.ENGLISH, getDisplayName(Locale) passing in itself as the locale, getLanguage(), and getISO3Language() respectively.